### PR TITLE
docs: add more args to appium-docs for publish

### PR DESCRIPTION
**Pull request by jlipps**
an attempt to address #1741 but it doesn't do the trick completely

### CODE REVIEW

**Comment by KazuCocoa** [comment]
oh, i see.
https://github.com/appium/appium-xcuitest-driver/actions/workflows/publish-doc.yml runs every release as https://github.com/appium/appium-xcuitest-driver/blob/master/.github/workflows/publish-doc.yml#L40-L42 to publish the latest doc since I haven't figured out the publish:docs's failure's cause yet. Maybe the task will be able to replace with this command eventually

File: package.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
     "dev": "npm run build -- --watch",
     "build:docs": "appium-docs build",
     "dev:docs": "appium-docs build --serve",
-    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --rebase",
+    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --rebase --remote origin --branch gh-pages --push",
```
Comment (boneskull) [inline package.json:101]: there is some brokenness in appium-docs which I am about to open a PR for. when that lands, please update to it. this should then work:

```suggestion
    "publish:docs": "appium-docs build --deploy -b docs-site -m 'docs: auto-build docs for appium-xcuitest-driver@%s' --push",
```

`--rebase` now defaults to `true`, `--remote` now defaults to `origin`, and `--branch` now defaults to `gh-pages` (if this wasn't true previously).  the only thing here that does _not_ default to `true` is `--push`.

PRO TIP: when using `implies` in yargs, if the depended-upon arg is a `boolean` and the default is not `true`, _you must be explicit_ about it and set `default: false`.

**Comment by boneskull** [comment]
see https://github.com/appium/appium/pull/18751

File: package.json (hunk 1 above)
Comment (jlipps) [reply package.json:101]: @boneskull this suggestion is essentially a revert, should we just close this PR instead?

**Pull request closed without merging**